python/res/enkf/config/gen_kw_config.py

```diff
@@ -113,3 +115,32 @@ def __eq__(self, other):
             return False
 
         return True
+
+    def getPriors(self):
```

Comment:
Lets keep new functions `snake_case`™️ Sorry for the confusing messages here, but the previous comment was a "typo"

Reply:
Ah missed that one